README.md

```diff
@@ -42,6 +42,7 @@ Table of contents
 - [License](https://github.com/kethinov/roosevelt#license)
 
 
+
```

Comment:
the new markdown editor i'm using inserts those newlines automatically. the document had previously been authored by a different editor